src/A2_Part6_Driver.cpp

```diff
@@ -6,16 +6,16 @@
 #include "Game/Game.h"
 
 /**
- *  //count the player with most elixers win 2 victory points.if tied then each receive 1
+ *  //count the player with most elixirs win 2 victory points.if Tied then each receive 1
```

Comment:
haha did you just do a find and replace for the word exilers and tied?

Reply:
lol yea tied it shouldn't have changed the comments. There's a method I changed from tied to Tied and it did that. Elixir I fixed spelling (so irrelevant, just me being thorough).